src/app/shared/cookies/browser-klaro.service.spec.ts

```diff
@@ -233,4 +260,46 @@ describe('BrowserKlaroService', () => {
       expect(ePersonService.patch).not.toHaveBeenCalled();
     });
   });
+
+  describe('initialize google analytics configuration', () => {
+    beforeEach(() => {
+      configurationDataService.findByPropertyName = findByPropertyName;
+      spyOn((service as any), 'getUser$').and.returnValue(observableOf(user));
+      translateService.get.and.returnValue(observableOf('loading...'));
+      spyOn(service, 'addAppMessages');
+      spyOn((service as any), 'initializeUser');
+      spyOn(service, 'translateConfiguration');
+    });
+    it('should have been initialized with googleAnalytics', () => {
+      const filteredServices = googleAnalyticsFilter(service);
+      expect(filteredServices.length).toBe(1);
+    });
+    it('should filter empty configuration', () => {
+      configurationDataService.findByPropertyName = jasmine.createSpy().and.returnValue(
+        createSuccessfulRemoteDataObject$({
+          ... new ConfigurationProperty(),
+          name: googleAnalytics,
+          values: [],
+        }));
+      service.initialize();
+      const filteredServices = googleAnalyticsFilter(service);
+      expect(filteredServices.length).toBe(0);
```

Comment:
Since we're filtering on name I think it's best to reflect that in these expectations.

Otherwise we could miss future issues where the filtering still _works_ but targets the wrong service → the length would be the same in that case!